### PR TITLE
chore(backport 1.3.x) backporting changes made for the argocdCD plugin on main into 1.3.x

### PR DESCRIPTION
**Pull request by Fortune-Ndlovu**
## Description

Backporting the updated changes made on the argoCD plugin, within the janus-idp main branch into the 1.3.x branch.

## Which issue(s) does this PR fix

https://issues.redhat.com/browse/RHIDP-3548 

## PR acceptance criteria

Please make sure that the following steps are complete:

- [x] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [x] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

File: dynamic-plugins/wrappers/backstage-community-plugin-redhat-argocd/package.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
+{
+  "name": "backstage-community-plugin-redhat-argocd",
+  "version": "1.7.0",
```
Comment (karthikjeeyar) [inline dynamic-plugins/wrappers/backstage-community-plugin-redhat-argocd/package.json:3]: the dependancy used here is "@backstage-community/plugin-redhat-argocd": "1.6.7" so can we keep that as is?

```suggestion
  "version": "1.6.7",
```

File: dynamic-plugins/wrappers/backstage-community-plugin-redhat-argocd/package.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  "scripts": {
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test --passWithNoTests --coverage",
+    "clean": "backstage-cli package clean",
+    "clean-dynamic-sources": "yarn clean && rm -Rf node_modules",
+    "export-dynamic": "janus-cli package export-dynamic-plugin --embed-package @backstage-community/plugin-redhat-argocd --override-interop default --in-place"
+  },
+  "dependencies": {
+    "@backstage-community/plugin-redhat-argocd": "1.6.7"
```
Comment (karthikjeeyar) [inline dynamic-plugins/wrappers/backstage-community-plugin-redhat-argocd/package.json:39]: The latest now is 1.8.0, I will create a followup PR to upgrade it in main branch and then backport it to 1.3.x.

**Comment by nickboldt** [comment]
Could you remove all the changes in this PR that are already done in another PR? For example the issue template stuff was merged in https://github.com/janus-idp/backstage-showcase/pull/1510/files and doesn't have anything to do with this PR.

**Comment by Fortune-Ndlovu** [comment]
cc: @nickboldt for review, please!

**Comment by nickboldt** [comment]
@karthikjeeyar back to you for updating the argocd versions in  1.3.x ^

**Pull request merged**